session.go

```diff
@@ -143,8 +215,12 @@ func (s *Session) createETWSession() error {
 		C.LPWSTR(unsafe.Pointer(&s.etwSessionName[0])),
 		pProperties,
 	)
-	if status := windows.Errno(ret); status != windows.ERROR_SUCCESS {
-		return fmt.Errorf("StartTrace failed; %w", status)
+	err := windows.Errno(ret)
+	if errors.Is(err, windows.ERROR_ALREADY_EXISTS) {
+		return ExistError{s.config.Name}
+	}
+	if !errors.Is(err, windows.ERROR_SUCCESS) {
+		return fmt.Errorf("StartTrace failed; %w", err)
 	}
```

Comment:
There is no errors wrapping here, so probably a simple switch would be more readable